### PR TITLE
Run integration testing workflow on multiple API levels

### PR DESCRIPTION
**Pull request by KacperKluka**
Used the Github Actions matrix strategy to run the `emulate` workflow on multiple API levels.

### CODE REVIEW

File: android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Build
+
+const val NOTIFICATION_CHANNEL_ID = "test-notification-channel"
+
+fun createNotificationChannel(context: Context) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```
Comment (QuintinWillison) [inline android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt:11]: Looks curious (zero!?). What does this achieve?

**Review by QuintinWillison — APPROVED** [review]

File: android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt (hunk 1 above)
Comment (KacperKluka) [reply android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt:11]: It's not zero but O as in Oreo :smile: This is needed as the channel creation method isn't backward compatible and calling it on devices with OS lower than Oreo will result in a runtime crash :wink:

Comment (QuintinWillison) [reply android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt:11]: That is hilarious, though perhaps worthy of a comment to clarify?

Comment (KacperKluka) [reply android-test-common/src/main/java/com/ably/tracking/test/android/common/NotificationHelpers.kt:14]: Yup, that's quite funny, but that's how we do things in the Android world :smile: Added the comment https://github.com/ably/ably-asset-tracking-android/pull/419/commits/8d66606295d2265d6bd7788848cec83bae4d4edd

**Pull request merged**